### DISCUSSION TITLE
ΕΞΕΤΑΣΗ ΜΑΘΗΜΑΤΟΣ

### DISCUSSION THREAD

**Discussion by DIATSIGK** [courses-ionio/sw#1008] [Announcements]
Καλησπέρα σας,
Ονομάζομαι ΔΙΑΤΣΙΓΚΟΣ ΔΗΜΗΤΡΙΟΣ (Α.Μ.: Π2018218) και είμαι Στ' εξάμηνο.
 Ειχα θέμα με το mail του ιονίου και δεν μπόρεσα να κάνω δήλωση μαθημάτων για την εξεταστική του εξαμήνου, ενώ αρχικά το είχα δηλώσει το μάθημα για να το παρακολουθήσω στο εξάμηνο(όπως και έκανα ). Θα μπορούσε να σταλέι κανονικά ο βαθμός στην γραμματεία; 
Σας είμαι υπόχρεος! 
Σας ευχαριστώ εκ των προτέρων!

**Comment by epidrome** [comment]
duplicate?
https://github.com/courses-ionio/sw/discussions/990#discussioncomment-801336
